.github/workflows/build_and_test.yml

```diff
@@ -419,3 +419,24 @@ jobs:
       run: |
         ./dev/change-scala-version.sh 2.13
         ./build/sbt -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Pkinesis-asl -Djava.version=11 -Pscala-2.13 compile test:compile
+
+  hadoop-2:
+    name: Hadoop 2 build
```

Comment:
We can upgrade from Hadoop 2.7.4 to Hadoop 2.10.x. So, the job name is only mentioning the major version.